content/fr/blog/gdpr-territorial-scope/index.md

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Portée Territoriale du RGPD",
+    "date": "2020-10-31T21:22:15+02:00",
+    "type": "blog",
+    "description": "Qui est habilité à faire valoir les droits des personnes concernées par le RGPD ? Et contre qui ? Voici un aperçu de la portée territoriale du RGPD.",
+    "featured_image": "gdpr-territorial-scope",
+    "tags": ["rgpd", "portée territoriale", "rgpd article 3" ],
+    "authors": [ "eliottha" ],
```

Comment:
Please add yourself to [this file](https://github.com/datenanfragen/website/blob/master/config/_default/params.toml), so we can list you as an author. You need to specify a short and a long name (the long name can optionally include a link to a website/social media profile/your GitHub account/etc. if you want).  
You of course don't need to use your real name, pseudonyms or user names are also absolutely fine.

Please also leave `"anikarini"` in the list as she originally wrote the post:

```suggestion
    "authors": [ "anikarini", "eliottha" ],
```

Reply:
By the way, feel free to also add yourself to the [`AUTHORS` file](https://github.com/datenanfragen/website/blob/master/AUTHORS) if you want. That way, you are listed on our [open source page](https://www.datarequests.org/open-source#contributors) as a contributor.

Reply:
Added myself, thank you!